js/swmp.js

```diff
@@ -310,4 +310,10 @@ disktable(jsondiskdata);
     $( function(){
       $( document ).trigger( "enhance.tablesaw" );
     });
-  })( jQuery );
\ No newline at end of file
+  })( jQuery );
+
+if(reload !== 0) {
```

Comment:
#6 this prevents the timer from showing up if you set it to 0. Not sure if I added a check for it anywhere else, though...